CI for Benchmarks fails

On Tioga we get errors like:

```
ld.lld: error: undefined symbol: clang::PCHContainerOperations::PCHContainerOperations()
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToIR()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
ld.lld: error: undefined symbol: clang::CompilerInstance::CompilerInstance(std::shared_ptr<clang::PCHContainerOperations>, clang::InMemoryModuleCache*)
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToIR()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
ld.lld: error: undefined symbol: clang::CompilerInstance::createDiagnostics(clang::DiagnosticConsumer*, bool)
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToIR()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
ld.lld: error: undefined symbol: clang::driver::Driver::Driver(llvm::StringRef, llvm::StringRef, clang::DiagnosticsEngine&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char>>, llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem>)
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToIR()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
ld.lld: error: undefined symbol: clang::driver::Driver::BuildCompilation(llvm::ArrayRef<char const*>)
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToIR()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
ld.lld: error: undefined symbol: clang::driver::Driver::ExecuteCompilation(clang::driver::Compilation&, llvm::SmallVectorImpl<std::pair<int, clang::driver::Command const*>>&)
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
ld.lld: error: undefined symbol: clang::CompilerInstance::~CompilerInstance()
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToDynamicLibrary()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
>>> referenced 3 more times
ld.lld: error: undefined symbol: clang::CompilerInvocationBase::CompilerInvocationBase()
>>> referenced by CppJitModule.cpp
>>>               CppJitModule.cpp.o:(proteus::CppJitModule::compileCppToIR()) in archive /usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install/lib64/libproteus.a
=> Execute make -j clean
rm -rf feynman-kac"-aot".x main"-aot".o
BUILD /usr/WS1/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/proteus-benchmarks/benchmarks/hecbench/hip/feynman-kac enable_proteus? True
Build command make -j CC=hipcc ENABLE_PROTEUS=yes PROTEUS_PATH=/usr/workspace/proteusdev/gitlab-runner/proteusdev/builds/PxDL3V6Bg/002/gitlab/proteus/proteus/build/install
```

and on lassen we get:

```
CMake Error at cmake/SetupLLVM.cmake:14 (find_package):
  Could not find a package configuration file provided by "Clang" with any of
  the following names:
    ClangConfig.cmake
    clang-config.cmake
  Add the installation prefix of "Clang" to CMAKE_PREFIX_PATH or set
  "Clang_DIR" to a directory containing one of the above files.  If "Clang"
  provides a separate development package or SDK, be sure it has been
  installed.
```

This manifested in PR #275

--- a/scripts/gitlab/ci-run-benchmarks.sh
+++ b/scripts/gitlab/ci-run-benchmarks.sh
@@ -44,12 +44,12 @@ fi
 
 echo "Install miniconda..."
 PYTHON_VERSION=3.12
-MINICONDA_DIR=miniconda3
+MINICONDA_DIR=/tmp/proteus-ci-${CI_JOB_ID}/miniconda3
 mkdir -p ${MINICONDA_DIR}
-wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(uname -m).sh -O ./${MINICONDA_DIR}/miniconda.sh
-bash ./${MINICONDA_DIR}/miniconda.sh -b -u -p ./${MINICONDA_DIR}
-rm ./${MINICONDA_DIR}/miniconda.sh
-source ./${MINICONDA_DIR}/bin/activate
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(uname -m).sh -O ${MINICONDA_DIR}/miniconda.sh
+bash ${MINICONDA_DIR}/miniconda.sh -b -u -p ${MINICONDA_DIR}
+rm ${MINICONDA_DIR}/miniconda.sh
+source ${MINICONDA_DIR}/bin/activate
 conda create -y -q -n proteus -c conda-forge \
     python=${PYTHON_VERSION} pandas==2.2.3 matplotlib==3.10.0
 conda activate proteus

